packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.service.js

```diff
@@ -32,8 +41,17 @@ export default class FlavorsList {
           region: currentRegion,
         }).$promise,
         prices: this.CucPriceHelper.getPrices(serviceName),
+        catalog: this.OvhApiMe.v6()
+          .get()
+          .$promise.then(
+            (me) =>
+              this.OvhApiOrderCatalogPublic.v6().get({
+                productName: 'cloud',
+                ovhSubsidiary: me.ovhSubsidiary,
```

Comment:
Related to my previous comment:
```suggestion
                ovhSubsidiary,
```

Reply:
This is done.  Thanks :)